diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 2c0480f0300731c636b62178b8b6d95d51605cda..1bd70fcdec0d379d6d855c65e9d48d8f47f949e6 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3468,6 +3468,7 @@ const FeatureEntry::FeatureVariation kFastCheckoutVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
