diff --git a/chrome/browser/resources/bookmarks/BUILD.gn b/chrome/browser/resources/bookmarks/BUILD.gn
index 222ebe0513c2ee4f7c06a5c4d333dc8e7157a323..cede8e8377a38745f755a8e94cef46832f97bf3d 100644
--- a/chrome/browser/resources/bookmarks/BUILD.gn
+++ b/chrome/browser/resources/bookmarks/BUILD.gn
@@ -71,4 +71,5 @@ build_webui("build") {
     optimize_webui_resource_paths_rewrites =
         [ "bookmarks.rollup.js|bookmarks.js" ]
   }
+  extra_build_grd_deps = [ "//brave/browser/resources/bookmarks:resources" ] extra_build_ts_in_files = brave_deps_chrome_browser_resources_bookmarks_in_files extra_build_ts_extra_deps = brave_deps_chrome_browser_resources_bookmarks_extra_deps
 }
