diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index e637213c2800134594ce22bc82a8cf3396f7bad7..899118529786ee4c7bed747645770099f6920291 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import {Route, Router} from './router.js';
 import {SettingsRoutes} from './settings_routes.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -246,6 +247,7 @@ function createBrowserSettingsRoutes(): Partial<SettingsRoutes> {
           loadTimeData.getString('performancePageTitle'));
     }
   }
+  addBraveRoutes(r);
   return r;
 }
 
