diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index 6ae327f77a3a9734d7f0bf76bce94315e2c29723..cd6ee3e590e088bcd30c8b9caf9ad1637875ba13 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -100,7 +100,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     if (router.getCurrentRoute() ===
-        (router.getRoutes() as {SYNC_ADVANCED: Route}).SYNC_ADVANCED) {
+        (router.getRoutes() as {BRAVE_SYNC_SETUP: Route}).BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   }
