diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index cf24caa550159ba8814395a8ddf0a2acda2232fe..2ada9ebc5ab244d93a039fa8296366b653289dcd 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -166,6 +166,7 @@ class ProfileImpl : public Profile {
   void OnLogin() override;
   void InitChromeOSPreferences() override;
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH)
+  BRAVE_PROFILE_IMPL_H_
 
   bool IsNewProfile() const override;
 
