diff --git a/chrome/browser/ui/startup/startup_tab_provider.cc b/chrome/browser/ui/startup/startup_tab_provider.cc
index de787141140f0bad02b682f3cf454382ad77ead1..9ff86d5a1c40a90fa36f6d8f699f39041db41241 100644
--- a/chrome/browser/ui/startup/startup_tab_provider.cc
+++ b/chrome/browser/ui/startup/startup_tab_provider.cc
@@ -319,6 +319,7 @@ StartupTabs StartupTabProviderImpl::GetPrivacySandboxTabs(
 bool StartupTabProviderImpl::CanShowWelcome(bool is_signin_allowed,
                                             bool is_child_account,
                                             bool is_force_signin_enabled) {
+  if ((true)) return true;
   return is_signin_allowed && !is_child_account && !is_force_signin_enabled;
 }
 
