diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index dad7560f0ef4aa76213f270121da78cff8f1b185..9127fa8fc73bda4c4c46d86a5473d18a7bad0604 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -492,4 +492,5 @@ build_webui("build") {
       "lazy_load.rollup.js|lazy_load.js",
     ]
   }
+  extra_build_ts_in_files = brave_deps_chrome_browser_resources_settings_in_files extra_build_ts_extra_deps = brave_deps_chrome_browser_resources_settings_extra_deps extra_build_grd_deps = [ "//brave/browser/resources/settings:resources" ]
 }
