diff --git a/chrome/browser/profiles/profile_shortcut_manager_win.cc b/chrome/browser/profiles/profile_shortcut_manager_win.cc
index 546ea17792d48a0cd5f1ad3c7d9b047c650430e8..d7b055c9c93c4d0fbc0df2701466f80e9b969349 100644
--- a/chrome/browser/profiles/profile_shortcut_manager_win.cc
+++ b/chrome/browser/profiles/profile_shortcut_manager_win.cc
@@ -859,7 +859,7 @@ bool ProfileShortcutManager::IsFeatureEnabled() {
 // static
 std::unique_ptr<ProfileShortcutManager> ProfileShortcutManager::Create(
     ProfileManager* manager) {
-  return std::make_unique<ProfileShortcutManagerWin>(manager);
+  return std::make_unique<BraveProfileShortcutManagerWin>(manager);
 }
 
 ProfileShortcutManagerWin::ProfileShortcutManagerWin(ProfileManager* manager)
