diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 74d355ac5e628420bd8f581a5b67970ab66a2497..d25ada0bfea4f9c8ae1ad4763ddee783a0d6fbf2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -1046,7 +1046,7 @@ public class ChromeTabbedActivity extends ChromeActivity<ChromeActivityComponent
                                     this, getModalDialogManager()));
             NotificationPermissionController.attach(
                     getWindowAndroid(), mNotificationPermissionController);
-            mNotificationPermissionController.requestPermissionIfNeeded(false /* contextual */);
+            if(false) mNotificationPermissionController.requestPermissionIfNeeded(false /* contextual */);
             if (BackPressManager.isEnabled()) initializeBackPressHandlers();
         }
     }
