diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7dd4f22f3d3fda81dce655cfb492f09a46596308..0d6f390a2ce5f41dd861cfb6ce427e9811c28c28 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2528,6 +2528,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
