diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index ccc7abb3ede8bacd914656903fee49d7f069e2a2..f72cd692540f87d5ca03812ef098a8f7ccfa1563 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -204,6 +204,7 @@ static_library("metrics") {
       "motherboard_metrics_provider.h",
     ]
   }
+  import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
 }
 
 if (is_android) {
