diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index f5e67107e5c39c0b031741d67676015c06690fb5..9a06d442723b8ea800506c7076359c8ab7a36669 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -333,6 +333,7 @@ void PermissionManager::RequestPermissionsFromCurrentDocument(
         permission_status_callback) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   RequestPermissionsInternal(permissions_types, render_frame_host,
                              requesting_origin, user_gesture,
@@ -383,6 +384,7 @@ PermissionManager::GetPermissionResultForCurrentDocument(
       PermissionUtil::PermissionTypeToContentSettingType(permission);
 
   const GURL requesting_origin =
+      BRAVE_FORCED_REQUESTING_ORIGIN
       PermissionUtil::GetLastCommittedOriginAsURL(render_frame_host);
   const GURL embedding_origin =
       GetEmbeddingOrigin(render_frame_host, requesting_origin);
