diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index 9d7be226373ab45b904c598e9e497da96116f394..8434f416548ba239a7d932d905931bb824b4a247 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -107,6 +107,7 @@ constexpr HistogramValue kHistogramValue[] = {
     {ContentSettingsType::NOTIFICATION_INTERACTIONS, 86},
     {ContentSettingsType::REDUCED_ACCEPT_LANGUAGE, 87},
     {ContentSettingsType::NOTIFICATION_PERMISSION_REVIEW, 88},
+    BRAVE_HISTOGRAM_VALUE_LIST
 };
 
 void FilterRulesForType(ContentSettingsForOneType& settings,
