diff --git a/ios/chrome/browser/prefs/browser_prefs.mm b/ios/chrome/browser/prefs/browser_prefs.mm
index b71669d952538ecd5a424697bfeb4b3fd95571f3..35bf5bf9d6e129a7b759dba22cd96bc8663490d0 100644
--- a/ios/chrome/browser/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/prefs/browser_prefs.mm
@@ -195,6 +195,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   // times a user should see autofill branding animation after installation.
   registry->RegisterIntegerPref(
       prefs::kAutofillBrandingIconAnimationRemainingCountPrefName, 2);
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -313,6 +314,7 @@ void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
 
   registry->RegisterDictionaryPref(kPrefPromoObject);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 
   // Register prerender network prediction preferences.
   registry->RegisterIntegerPref(
