diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 902a3b8d06fc1d3b5138c3e50251e3a4818b37b6..780fccf5b13b36fff5106c8daef4adb5abb32a02 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6292,3 +6292,4 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+from import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
