diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index 62dae0d6e8bdb14e187fe51b9c68fefdc7b717ba..4c9303e0898defb0e714d2a357657f40a6bec585 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -677,6 +677,7 @@ PendingScript* ScriptLoader::PrepareScript(
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior,
                              RejectCoepUnsafeNone(false));
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="28">Let settings object be el's node document's relevant
   // settings object.</spec>
